CONTRIBUTING.md

```diff
@@ -60,50 +63,26 @@ All changes to the `main` branch will be automatically deployed via
 
 ## Translations
 
-Juicebox uses [`lingui.js`](https://lingui.js.org) for internationlization
-(i18n). Languages we support are defined in `.linguirc.json`. `en` is our source
-language.
+Juicebox uses [Crowdin](https://crowdin.com/project/juicebox-interface) for
```

Comment:
**praise**: great doc updates!

It would be worth describing the technical details a bit more. If you were to be hit by a bus (hey, it's happened more than once right), a new dev should be able to come here (or another document) and learn everything they need to.

Don't think it's blocking for this PR. Might be a nice follow-up :)

Reply:
What other technical details could we add that could be helpful for a new developer outside of describing how the Github actions are triggered?